package.json

```diff
@@ -388,6 +388,7 @@
     "react-table": "7.8.0",
     "react-transition-group": "4.4.5",
     "react-use": "17.4.0",
+    "react-virtual": "2.8.2",
```

Comment:
This is already a transitive dependency, as kbar depends on it. Because we've moved a kbar component into our own code, we must now explicitly list this here, but it doesn't add to bundle size. 

We'll remove it once kbar supports this directly.